src/main/java/com/wavefront/helpers/Defaults.java

```diff
@@ -43,5 +43,9 @@
 
   public static final List<ValueDistribution> DEFAULT_SPANS_DISTRIBUTIONS = List.of(
       new ValueDistribution(3, RANDOM.nextInt(10) + 5, 100));
+  public static final List<String> STAT_ROOT_NAME_LIST = List.of(
```

Comment:
I guess that TraceTypePatternValidator should be updated to prohibit the usage of these words as a service name.